src/nvim/globals.h

```diff
@@ -1123,7 +1123,7 @@ EXTERN char_u e_noalt[] INIT(= N_("E23: No alternate file"));
 EXTERN char_u e_noabbr[] INIT(= N_("E24: No such abbreviation"));
 EXTERN char_u e_nobang[] INIT(= N_("E477: No ! allowed"));
 EXTERN char_u e_nogvim[] INIT(= N_(
-        "E25: GUI cannot be used: Not enabled at compile time"));
+        "E25: Nvim does not have built-in GUI"));
```

Comment:
Line fits in one line. The corresponding update to `runtime/doc/message.txt`, line 318, is missing.